openpype/tools/loader/widgets.py

```diff
@@ -614,26 +615,30 @@ def on_context_menu(self, point):
             box.show()
 
     def group_subsets(self, name, asset_ids, items):
-        field = "data.subsetGroup"
+        subset_ids = {
+            item["_id"]
+            for item in items
+            if item.get("_id")
+        }
+        if not subset_ids:
+            return
```

Comment:
Does this also work when multi-selecting (multiple assets) in the loader left hand side and then trying to group on the subsets? I feel like the old code might have been that way to support that case - but I'm not sure?

Reply:
Good point, testing.

Reply:
Yes it work as expected, and found other unrelated bug :)